api/v1beta1/common_types.go

```diff
@@ -53,3 +66,49 @@ type PasswordSelector struct {
 	// Service - Selector to get the watcher service user password from the Secret
 	Service string `json:"service"`
 }
+
+type WatcherImages struct {
+	// +kubebuilder:validation:Required
+	// APIContainerImageURL
+	APIContainerImageURL string `json:"apiContainerImageURL"`
+
+	// +kubebuilder:validation:Required
+	// DecisionEngineContainerImageURL
+	DecisionEngineContainerImageURL string `json:"decisionengineContainerImageURL"`
+
+	// +kubebuilder:validation:Required
+	// DecisionEngineContainerImageURL
```

Comment:
This should be ApplierContainerImageURL ?

Reply:
Done!